.projectile

```diff
@@ -0,0 +1 @@
+-/vendor
```

Comment:
I added it because I think of it as similar to a `.gitignore` file which can often contain subjectively-named file paths. e.g. https://github.com/puppetlabs/beaker-hostgenerator/blob/master/.gitignore#L13-L14 (That `.gitignore` originally came from puppetlabs/beaker-template)

The difference here is that I don't care as much about `.gitignore` so I never bothered to add `vendor` to it the way I do for projectile. Local package installation is much more annoying in projectile than in git.